src/longopt/grdimage_inc.h

```diff
@@ -42,7 +42,7 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = { /* Local options for this m
                   "a,d,m,n",           "azimuth,default,ambient,intensity" },
 	{ 0, 'M', "monochrome",        "", "", "", "" },
 	{ 0, 'N', "noclip",            "", "", "", "" },
-	{ 0, 'Q', "transvalue",
+	{ 0, 'Q', "transvalue|alphacolor",
```

Comment:
``transvalue`` is not nice. Again, Julia uses ``alpha_color``, ``nan_alpha``, PyGMT use ``nan_transparent``.